src/persist/src/nemesis/generator.rs

```diff
@@ -51,10 +51,7 @@ impl GeneratorConfig {
 
 impl Default for GeneratorConfig {
     fn default() -> Self {
-        let mut ops = Self::all_operations();
-        // TODO: Re-enable this once it's not flaky.
-        ops.allow_compaction_weight = 0;
-        ops
+        Self::all_operations()
```

Comment:
Lets set this up to skip the thrashing between >0 things are disabled and 0 things are disabled. Something like:

```rust
#[allow(unused_mut)]
let mut ops = Self::all_operations();
// NB: If we need to temporarily disable an operation in all the nemesis tests, set it to 0 here. (As opposed to clearing it in the impl of `all_operations`, which will break the Generator tests.)
ops
```